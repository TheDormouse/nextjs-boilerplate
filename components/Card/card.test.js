import ReactDOM from 'react-dom';
import { Card } from './';
import bg from '../../public/bg.jpg'

it('Matches snapshot', () => {
  const div = document.createElement('div');
  ReactDOM.render(
   <Card src={bg} title='Jest Testing' />,
    div
  );
  expect(div).toMatchSnapshot();

  ReactDOM.unmountComponentAtNode(div);
});