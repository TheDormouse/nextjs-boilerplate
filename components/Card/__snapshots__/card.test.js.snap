// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
    >
      <div
        style="box-sizing: border-box; display: block; max-width: 100%;"
      >
        <img
          alt=""
          aria-hidden="true"
          role="presentation"
          src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjI1MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
          style="max-width: 100%; display: block;"
        />
      </div>
      <img
        alt="Some card"
        class="card"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
      />
    </div>
    <p
      class="text"
    >
      Jest Testing
    </p>
  </div>
</div>
`;
